### ISSUE TITLE
Linkedin Dark awesome, but not working on linkedin sales navigator pages

### ISSUE THREAD

**Issue by jamesluffman** [Jordy3D/Jordy3D.github.io#3]
Hi Jordy this is great, saving my eyes on Linkedin!

Could you make it work on the linkedin sales navigator pages? Right now they are still mostly white

**Comment by Jordy3D** [comment]
Ah, that's one place that I could never access (since I lack Premium)
I'll start my Free Trial when I get a chance in the next few days and go through it.

Thanks for letting me know I missed stuff though! 
There's some minor changes and updates I've made since the last push that will go live soon as well.

**Comment by Jordy3D** [comment]
Pushed an update that should cover at least most of Sales Navigator. Let me know if there's any missing points (I have rather limited time to update it before the Trial ends)
